.dependabot/config.yml

```diff
@@ -7991,15 +8123,6 @@ update_configs:
   - package_manager: "dotnet:nuget"
     directory: "/snippets/standard/datetime/calendars/specify-era/vb" # vb.vbproj
     update_schedule: "live"
-  - package_manager: "dotnet:nuget"
-    directory: "/snippets/standard/datetime/json/datetime-converter-examples/example1" # example1.csproj
-    update_schedule: "live"
-  - package_manager: "dotnet:nuget"
-    directory: "/snippets/standard/datetime/json/datetime-converter-examples/example2" # example2.csproj
-    update_schedule: "live"
-  - package_manager: "dotnet:nuget"
-    directory: "/snippets/standard/datetime/json/datetime-converter-examples/example3" # example3.csproj
-    update_schedule: "live"
```

Comment:
These are examples of wrong directories that dependabot didn't open issue for them.

/cc @richlander